apps/src/blockly/customBlocks/googleBlockly/mutators/procedureDefMutator.js

```diff
@@ -0,0 +1,228 @@
+/**
+ * This is a direct copy of `procedureDefMutator` from @blockly/block-shareable-procedures
+ * with the compose() and decompose() methods removed. These methods automatically
+ * add a gear icon UI that we do not want. A future version of the plugin will
+ * export this mutator (and other extensions), but this will require bumping to
+ * Blockly v10.
+ * TODO: Once we are on Blockly v10, remove this file.
```

Comment:
Should I go ahead and add a ticket for this to the Code Tools backlog (potentially pending new board creation)?

Reply:
That would be great, thank you!